challengeMk2/ViewModels/SystemDetailViewModel.cs

```diff
@@ -91,9 +91,9 @@ async void UpdateSystemData()
             //2° Get distance and number of bodies from currentSystem and put them in detailedSystem
             GetCompInfos();
 
-        }
-
-         async Task<StarSystem> GetDetailsFromApi()
+        }
+
+        async Task<StarSystem> GetDetailsFromApi()
         {
             HttpClient client = new HttpClient();
```

Comment:
Don't forget to use the `using` keyword to dispose the client after having used it.

Reply:
Sorry, it is my fault, this class should be deleted. 
It is replaced by SystemDetailCarouselViewModel.cs and not be used anymore.